src/storybook/stand-alone-documentaion/accessibility/principles/principlesAccessibility.jsx

```diff
@@ -1,11 +1,9 @@
-import { principleClarity, principleSpeed, principleIntuative, principleDelight } from "../assets";
+import { Flex } from "components";
+import { principleClarity, principleDelight, principleIntuative, principleSpeed } from "../assets";
 import { Principle } from "../../welcome/principle/principle";
-import "../../welcome/principles/principles.scss";
 
-const BASE_CLASS = "monday-storybook-principles";
-
-export const PrinciplesAccesability = () => (
-  <div className={BASE_CLASS}>
+export const PrinciplesAccessibility = () => (
+  <Flex direction={Flex.directions.ROW} wrap gap={Flex.gaps.LARGE} align={Flex.align.START}>
     <Principle
       imgSrc={principleClarity}
```

Comment:
I am not sure that in this case flex wrap will be the right approach.  Check it with meirav please